File: PooCC01CNetz/Semana04/Java/Ejercicios02/Domicilio.java
public class Domicilio {
	private String calle;
	private int num;
	private int cp;
	
	public Domicilio(String calle, int num, int cp) {
		this.calle = calle;
		this.num = num;
		this.cp = cp;
	}
	public Domicilio() {}
	
	public void setCalle(String calle) {
		this.calle = calle;
	}
	public String getCalle() {
		return calle;
	}
	public void setNum(int num) {
		this.num = num;
	}
	public int getNum() {
		return num;
	}
	public void setCp(int cp) {
		this.cp = cp;
	}
	public int getCp() {
		return cp;
	}
	
	public void imprimeDireccion() {
		System.out.println("Calle: " + calle);
		System.out.println("Num  : " + num);
		System.out.println("CP   : " + cp);
	}
}

File: PooCC01CNetz/Semana04/Python/Ejercicios02/Domicilio.py
class Domicilio:
    def __init__(self, calle=None, num=None, cp=None):
        # Constructor vacio
            # Atributos privados
        if calle is None and num is None and cp is None:
            self.__calle = calle
            self.__num = num
            self.__cp = cp
        else:
            # Constructor con parámetros
            self.__calle = calle
            self.__num = num
            self.__cp = cp
    
    def setCalle(self, calle):
        self.__calle = calle

    def getCalle(self):
        return self.__calle
    
    def setNum(self, num):
        self.__num = num

    def getNum(self):
        return self.__num
    
    def setCp(self, cp):
        self.__cp = cp

    def getCp(self):
        return self.__cp
    
    def imprimeDomicilio(self):
        print("Calle    : " + self.__calle)
        print("Numero   : " + self.__num)
        print("CP       : " + self.__cp)